commands/git/git-commit.md

```diff
@@ -0,0 +1,25 @@
+### Commit
+
+
+#### Commit the changes
+
+`git commit -m {{MESSAGE}}`
+
+- <b>MESSAGE: </b> The message on the commmit
+
+##### Example:
+
+`git commit -m "Changed the name"`
+
+
+
+#### Push the changes
```

Comment:
Can you change this to separate file too?
We want a single command in each file.